src/composables/useNumbers.ts

```diff
@@ -62,5 +90,67 @@ export default function useNumbers() {
     return tokenAmount.times(price).toString();
   }
 
-  return { fNum, toFiat };
+  function fNum2(
+    number: number | string,
+    options: FNumOptions | undefined = {}
+  ): string {
+    if (typeof number === 'string') {
+      number = number || 0;
+      number = new BigNumber(number).toNumber();
+    }
+
+    const formatterOptions: Intl.NumberFormatOptions = Object.assign(
+      {},
+      options
+    );
```

Comment:
nit: is it possible to just spread it here to make it more concise (unless there is a reason you are using obj.assign)